config/initializers/rack_attack.rb

```diff
@@ -8,9 +8,14 @@ class Attack
       IPAddr.new(x)
     end.freeze
 
-    EMAIL_REGISTRATION_PATHS = ['/sign_up/enter_email', '/en/sign_up/enter_email',
-                                '/es/sign_up/enter_email', '/fr/sign_up/enter_email'].freeze
-    SIGN_IN_PATHS = ['/', '/en', '/es', '/fr'].freeze
+    EMAIL_REGISTRATION_PATHS =
+      IdentityConfig.store.available_locales.reduce(['/sign_up/enter_email']) do |paths, locale|
+        paths << "/#{locale}/sign_up/enter_email"
+      end.freeze
+
+    SIGN_IN_PATHS = IdentityConfig.store.available_locales.reduce(['/']) do |paths, locale|
+      paths << "/#{locale}"
+    end.freeze
```

Comment:
Weighing whether something like `[nil, *IdentityConfig.store.available_locales].map` could work with including the unprefixed default (to avoid `reduce` and repeated path string), but can't think of an especially elegant implementation that'd be better than what you came up with here.

Reply:
I have a soft spot for `reduce`, but I'm definitely not committed to it. If your suggestion is clearer, I will switch to that.

Reply:
Not a blocker, I don't have any real alternative to propose 👍